java/semantickernel-api/src/main/java/com/microsoft/semantickernel/tokenizers/settings/EmbeddedResource.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft. All rights reserved.
+package com.microsoft.semantickernel.tokenizers.settings;
+
+import java.io.*;
```

Comment:
Please import fully qualified classes instead of *. I've got no idea what you are using from java.io at this point. If your IDE is doing this automagically, you can adjust the setting.